tests/components/climate/test_demo.py

```diff
@@ -237,6 +237,13 @@ def test_set_aux_heat_bad_attr(self):
         self.hass.block_till_done()
         self.assertEqual('off', state.attributes.get('aux_heat'))
 
+    def test_set_aux_heat_on_ecobee(self):
+        """Test setting the auxillary heater on/true."""
+        climate.set_operation_mode(self.hass, 'auxHeatOnly', ENTITY_CLIMATE)
```

Comment:
I think this needs to be `ENTITY_ECOBEE` as the climate entity does not have `auxHeatOnly`